code/game/gamemodes/battleroyale/royale.dm

```diff
@@ -67,9 +67,11 @@
 /datum/game_mode/battle_royale/special_report()
     if(winner == "draw")
         to_chat(world, "<span class='ratvar'><font size=12>Everybody died!</font></span>")
+        sendooc2tgs("**Everybody died!**\n--------------------------------------")
         return "<div class='panel redborder'><span class='redtext big'>Nobody claims victory!</span></div>"
     if(winner?.real_name)
         to_chat(world, "<span class='ratvar'><font size=12>[winner.real_name] claims victory!</font></span>")
+        sendooc2tgs("**[winner.real_name] has claimed victory!**\n--------------------------------------")
```

Comment:
Splitting this into two messages is bad, it looks bad. If you can find a way that doesn't involve sending two messages at roundend that would be great

Reply:
yeah, definitely will be trying to now that I've gotten the gist of using it.